src/functions_framework/event_conversion.py

```diff
@@ -143,6 +160,74 @@ def background_event_to_cloudevent(request) -> CloudEvent:
     return CloudEvent(metadata, data)
 
 
+def is_convertable_cloud_event(request) -> bool:
+    """Is the given request a known CloudEvent that can be converted to background event."""
+    if is_binary(request.headers):
+        event_type = request.headers.get("ce-type")
+        event_source = request.headers.get("ce-source")
+        return (
+            event_source is not None
+            and event_type is not None
+            and event_type in _CE_TO_BACKGROUND_TYPE
+        )
+    return False
+
+
+def _split_ce_source(source) -> Tuple[str, str]:
+    """Splits a CloudEvent source string into resource and subject components."""
+    regex = re.compile(r"\/\/([^/]+)\/(.+)")
+    match = regex.fullmatch(source)
+    if not match:
+        raise EventConversionException("Unexpected Cloud Event source.")
+
+    return match.group(1), match.group(2)
+
+
+def cloud_event_to_background_event(request) -> Tuple[Any, Context]:
```

Comment:
Nit: `cloudevent`, since it's stylized as CloudEvent and the other conversion function uses `cloudevent`